src/drivers/safety_button/SafetyButton.cpp

```diff
@@ -190,25 +154,9 @@ SafetyButton::Run()
 		return;
 	}
 
-	CheckButton();
-
-	// control safety switch LED & publish safety topic
-	if (!PX4_MFT_HW_SUPPORTED(PX4_MFT_PX4IO)) {
```

Comment:
Did you check this is handled correctly?

Reply:
updated this to:
```
	if (!_has_px4io) {
		FlashButton();
		CheckSafetyRequest(button_pressed);
	}
```

Reply:
additionally tested on Pixracer with Safety Button. Led on button blinks and button works.